src/routes/docs/products/storage/quick-start/+page.markdoc

```diff
@@ -119,7 +119,34 @@ To upload a file, add this to you web, Flutter, Apple, or Android app.
 
   --cec8e8123c05ba25--
   ```
+  ```client-react-native
+  import { Client, Storage, ID, Permission } from 'react-native-appwrite';
 
+  const client = new Client()
+      .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+      .setProject('<PROJECT_ID>');                 // Your project ID
+
+  const storage = new Storage(client);
+
+  const promise = await storage.createFile(
+      '[BUCKET_ID]',
+      ID.unique(),
+      { 
+      name: 'file.jpeg', 
+      type: 'image/jpeg', 
+      uri: 'file:///data/user/0/host.exp.exponent/cache/ExperienceData/text.jpeg', 
```

Comment:
Also make sure the URI is something generic and reasonable :)

Reply:
Also ask @lohanidamodar to see if there's `inputfiles` for react native.

Reply:
Confirmed with @lohanidamodar that there are currently no `inputfiles` for react native